tests/lib/rules/require-render-return.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview Enforces render function to always return value.
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/require-render-return')
+const RuleTester = require('eslint').RuleTester
+
+const parserOptions = {
+  ecmaVersion: 6,
+  sourceType: 'module',
+  ecmaFeatures: { experimentalObjectRestSpread: true, jsx: true }
+}
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester()
+ruleTester.run('require-render-return', rule, {
+  valid: [
+    {
+      code: `Vue.component('test', {
+        render() {
+          return {}
+        }
+      })`,
+      parserOptions
+    },
+    {
+      code: `Vue.component('test', {
+        render() {
+          return <div></div>
+        }
+      })`,
+      parserOptions
+    },
+    {
+      filename: 'test.vue',
+      code: `export default {
+        render() {
+          return {}
+        }
+      }`,
+      parserOptions
+    },
+    {
+      filename: 'test.vue',
+      code: `export default {
+        render: () => null
+      }`,
+      parserOptions
+    }
```

Comment:
Could you add tests that this rule does not report functions which are not the `render`?